tools/testsys/src/crds.rs

```diff
@@ -166,6 +168,42 @@ impl<'a> CrdInput<'a> {
             "conformance-registry".to_string() => some_or_null(&self.config.conformance_registry),
         }
     }
+
+    /// Find the crd template file for the given test type
+    fn custom_crd_template_file_path(&self) -> Option<PathBuf> {
+        let test_type = &self.test_type.to_string();
+        // List all acceptable paths to the custom crd to allow users some freedom in the way
+        // `tests` is organized.
+        let acceptable_paths = vec![
+            // Check for <TEST-TYPE>.yaml in the top level directory
+            PathBuf::new().join(test_type).with_extension("yaml"),
+            // Check for <TESTSYS_FOLDER>/<TEST-TYPE>/<TEST-TYPE>.yaml
+            self.tests_directory
+                .join(test_type)
+                .join(test_type)
```

Comment:
We don't want `.join(test_type)` twice, do we?

Reply:
We do. This allows putting the test yaml at `<TESTSYS_FOLDER>/<TEST-TYPE>/<TEST-TYPE>.yaml`

Reply:
Ah, sorry, I see it now.